src/Microsoft.VisualStudio.ProjectSystem.IntegrationTests/Utilities/FeatureAttribute.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.IntegrationTests
+{
+    internal partial class FeatureAttribute
+    {
+        public const string AutomaticEndConstructCorrection = nameof(AutomaticEndConstructCorrection);
```

Comment:
Any reason to keep this in sync with Roslyn? Otherwise we can get rid of a bunch of these I assume.